ERCS/erc-XXXX.md

```diff
@@ -0,0 +1,304 @@
+---
+eip: XXXX
```

Comment:
```suggestion
eip: 8002
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.